packages/modules/src/modules/epics/ContributionsEpicTicker.tsx

```diff
@@ -75,7 +75,7 @@ const filledProgressStyles = (end: number, runningTotal: number, total: number):
         bottom: 0;
         transform: ${progressBarTransform(end, runningTotal, total)};
         transition: transform 3s cubic-bezier(0.25, 0.55, 0.2, 0.85);
-        background-color: ${palette.brandAlt[400]};
+        background-color: #c70000;
```

Comment:
not very important but I think this can be `news[400]`
https://github.com/guardian/source/blob/bae4c766e12b89243cf78033f8d621312d595dc4/packages/%40guardian/source-foundations/src/palette.ts#L13